.travis.yml

```diff
@@ -12,7 +12,12 @@ env:
     - DISTRO=debian-stable
     - PACKAGE=nbd
   matrix:
-    - BASE_REMOTE=git://github.com/xapi-project/xs-opam
+    # We need to pass some Travis environment variables to the container to
+    # enable uploading to coveralls and detection of Travis CI.
+    # Also, set TESTS to false to avoid running them twice.
+    - BASE_REMOTE=git://github.com/xapi-project/xs-opam \
+      TEST=false \
+      POST_INSTALL_HOOK="env TRAVIS=$TRAVIS TRAVIS_JOB_ID=$TRAVIS_JOB_ID COVERAGE=1 bash -ex .coverage.sh"
```

Comment:
I don't think that we need `COVERAGE=1` here if it's also set in `.coverage.sh`.